inc/class-ad-layers-admin.php

```diff
@@ -82,7 +82,7 @@ public function enqueue_scripts() {
 					get_ad_layers_path( 'adLayersAdmin.js' ),
 					get_ad_layers_dependencies( 'adLayersAdmin.php' ),
 					get_ad_layers_hash( 'adLayersAdmin.js' ),
-					true
+					false
```

Comment:
Maybe this one doesn't need to be in the head?  Looks like it's admin and, if the comment is to be believed, just styles.

On the other hand... better safe than sorry?  Probably couldn't hurt to enqueue in it in the head even if it doesn't strictly need to.